.NET/Microsoft.Recognizers.Text.Sequence/Chinese/Extractors/ChineseCreditCardExtractorConfiguration.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Definitions.Chinese;
+
+namespace Microsoft.Recognizers.Text.Sequence.Chinese
```

Comment:
As all cards should be recognized in all languages. It's preferable to move the basic implementation to English, and make all languages fallback to it, unless they need special config. Similar to how URL works.